integrity-backend/input-chatbot-examples/generic-meta-content.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contentMetadata":
+  {
+    "name": "Chat archive bot",
```

Comment:
These names will be tough.... I dont know where to pull them from.

Reply:
Got a bit creative for bot:
https://github.com/starlinglab/integrity-preprocessor/blob/schema/chat-bot/main.py#L76-L79

Browertrix and dropbox i have a constant
https://github.com/starlinglab/integrity-preprocessor/blob/schema/browsertrix/main.py#L106-L111

https://github.com/starlinglab/integrity-preprocessor/blob/preprocessor_dropbox/dropbox/main.py#L29-L34

Reply:
Can we extract from filename?

```
"uploadFilename": "John Snow__My Archive.wacz"
```

Reply:
I wonder if author is constant or whoever captured. Maybe using Hala as author, and then having actual user info in private is safer.

Reply:
>    ] = f"Archive of chat by {bot_type} bot on starting on {cosmetic_date}"

This sentence is wrong. See my PR.